sbt-bridge/test/xsbt/ExtractUsedNamesSpecification.scala

```diff
@@ -306,7 +306,7 @@ class ExtractUsedNamesSpecification {
     // All classes extend Object
     "Object",
     // All classes have a default constructor called <init>
-    "java.lang.Object;init;",
+    "java::lang::Object;init;",
```

Comment:
It might be useful to use `;` as a separator like in https://github.com/sbt/zinc/pull/288 so that builds involving both scala2 and scala3 projects have incremental compilation working correctly? (Which makes me wonder if https://github.com/sbt/zinc/pull/288 is missing a change in the java api extractor)